package.json

```diff
@@ -3,19 +3,26 @@
   "version": "6.0.0",
   "description": "a collection of utility functions for Ethereum",
   "main": "dist/index.js",
+  "types": "./dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "coverage": "npm run build:dist && istanbul cover _mocha",
+    "build": "tsc -p ./tsconfig.prod.json",
+    "prepublishOnly": "npm run test && npm run build",
+    "coverage": "npm run build && istanbul cover _mocha",
     "coveralls": "npm run coverage && coveralls <coverage/lcov.info",
```

Comment:
Can you also use [ethereumjs-config-nyc](https://github.com/ethereumjs/ethereumjs-config/tree/master/packages/ethereumjs-config-nyc)?

Reply:
This library uses mocha for testing and tests against the `dist/` files, and further runs karma. I tried to make it work with the nyc scripts with no luck so far. I think it would require larger changes, and probably would need to be tested against `src/` files for coverage with `ethereumjs-config-nyc` configs to work. Should I attempt the changes in this PR?

Reply:
Ah no, then leave for now. Sorry, didn't have a close-enough look.